### ISSUE TITLE
Buildpack detection issue - Java Main, Spring Boot CLI

### ISSUE THREAD

**Issue by helenwilliamson** [cloudfoundry/java-buildpack#119]
Hiya,

I'm trying to deploy a scala unfiltered, netty app to cloud foundry and I'm having problems with buildpacks. 

When I try to push the app, I get the following error message:
```
2014-12-12T15:01:42.50+0000 [STG]     ERR [Buildpack]                      ERROR Compile failed with exception #<RuntimeError: Application can be run by more than one container: Java Main, Spring Boot CLI>
2014-12-12T15:01:42.50+0000 [STG]     ERR Application can be run by more than one container: Java Main, Spring Boot CLI
```
The project is [https://github.com/helenwilliamson/simple-service]

I'm expecting this project to be built using Java Main, not Spring Boot CLI.  Unfortunately one of the dependencies (logback-classic) contains one .groovy file (EvaluatorTemplate). 

From looking at the defintion of spring_boot_cli.rb, this buildpack will be chosen if there is 1 or more .groovy files in the project, they're all POGOs, none contain a main method or shebang and there's no WEB-INF directory. 

Unfortunately netty apps don't have a WEB-INF. Is the POGO detection correct as the class it's picking up doesn't seem to be a POGO (it implements one interface with one method)

My problem seem kind of similar to https://github.com/cloudfoundry/java-buildpack/issues/24, except that I'm guessing it is valid to have .class files in a Spring Boot CLI project.

Thanks, Helen

**Comment by cgfrost** [comment]
Hi Helen,
I'm spending some time to look at this issue now. It's been a while since it was raised so I'm wondering if you have found a work around or had any ideas. In the mean time I have your test application am working to reproduce the issue.
Chris.

**Comment by stephen-souness-springer** [comment]
I suspect that the underlying cause of this may be due to logback including a .groovy file.

**Comment by nebhale** [comment]
Resolved in 623c428.
